core/gui/src/app/dashboard/component/user/user-dataset/user-dataset.component.ts

```diff
@@ -71,7 +70,7 @@ export class UserDatasetComponent implements AfterViewInit {
       .subscribe(() => this.search());
   }
 
-  async search(forced: Boolean = false): Promise<void> {
+  async search(forced: Boolean = false, filterScope: "all" | "public" | "private" = "all"): Promise<void> {
```

Comment:
how will this filterScope being set? Please add some comments, i.e. when it will be set as all, public or private.

Reply:
Ditto